src/app/modules/app.server.module.ts

```diff
@@ -4,23 +4,17 @@ import { ServerModule } from '@angular/platform-server';
 import { AppModule } from './app.module';
 import { AppComponent } from '../app.component';
 
-import { AuthClientService } from '../services/auth.client.service';
-import { AuthServerService } from '../services/auth.server.service.service';
-import { AuthConfigService } from '@auth0/auth0-angular';
+import { AuthConfigService, AuthService } from '@auth0/auth0-angular';
 
 @NgModule({
-  imports: [
-    AppModule,
-    ServerModule
-  ],
-  providers:[
-    AuthServerService,
+  imports: [AppModule, ServerModule],
+  providers: [
     {
-      provide: AuthClientService,
-      useClass: AuthServerService,},
-      {provide:AuthConfigService,
-      useClass:{} as any}
+      provide: AuthService,
+      useValue: {},
```

Comment:
This is all you need to mock our entire AuthService. Ofcourse, this might fail if some methods are accessed on the server. If that's the case, you need to bring back some kind of mock implementation like you had with AuthServerService.

But for your sample, this didn't seem necessary.